RULES.md

```diff
@@ -81,6 +81,7 @@ Each Notice is associated with a severity: `INFO`, `WARNING`, `ERROR`.
 | [`stop_time_timepoint_without_times`](#stop_time_timepoint_without_times)                                                         | `arrival_time` or `departure_time` not specified for timepoint.                                                                                        |
 | [`stop_time_with_arrival_before_previous_departure_time`](#stop_time_with_arrival_before_previous_departure_time)                 | Backwards time travel between stops in `stop_times.txt`                                                                                                |
 | [`stop_time_with_only_arrival_or_departure_time`](#stop_time_with_only_arrival_or_departure_time)                                 | Missing `stop_times.arrival_time` or `stop_times.departure_time`.                                                                                      |
+| [`stop_without_location`](#stop_without_location)                                                                                   | `stop_lat` and/or `stop_lon` missing for strop with `location_type` of `0`, `1`, or `2`.                                                                                                                |
```

Comment:
nit: is the table alignment off here?

Reply:
It actually doesn't look off to me. The lines are so wide, you have to stretch the editor window full screen to see alignment.  Here's the 10,000 foot view:
<img width="2143" alt="image" src="https://user-images.githubusercontent.com/40084/218267639-23f3e1af-ffdd-419a-8645-29e371315b96.png">